nevergrad/optimization/test_optimizerlib.py

```diff
@@ -407,29 +407,30 @@ def test_parallel_es() -> None:
 @pytest.mark.parametrize(
     "dimension, num_workers, scale, budget, ellipsoid",
     [
-    (2, 3, 1., 120, False),
-    #(2, 1, 8., 120, True),
-    (2, 3, 8., 70, False),
-    #(1, 1, 1., 20, True),
-    (1, 3, 5., 20, False),
-    #(2, 3, 1., 70, True),
-    (2, 1, 8., 40, False),
-    #(2, 3, 8., 70, True),
-    (5, 1, 1., 150, False),
-    #(5, 3, 1., 225, True),
-    (5, 1, 8., 150, False),
-    #(5, 3, 8., 500, True),
-    (8, 27, 8., 200, True),
-    (9, 27, 8., 700, True),
-    #(10, 27, 8., 400, False),
+        # (2, 3, 1., 120, False),
```

Comment:
@teytaud I deactivated this test which was flaky, I am a bit concerned that all of them may be. If so they must be improved or removed

Reply:
Solved in another PR.